README.md

```diff
@@ -29,46 +29,115 @@ since there may be undiscovered or unresolved issues at HEAD.
 From a Linux or Mac OS X client:
 
 ```
-$ git clone https://github.com/kubernetes/deployment-manager.git
-$ cd deployment-manager
+$ git clone https://github.com/kubernetes/helm.git
+$ cd helm
 $ make build
 $ bin/helm server install
 ```
 
-That's it. You can now use `kubectl` to see DM running in your cluster:
+That's it. You can now use `kubectl` to see Helm running in your cluster like this:
 
 ```
-kubectl get pod,rc,service --namespace=dm
+$ kubectl get pod,rc,service --namespace=dm
+NAME                    READY        STATUS        RESTARTS   AGE
+expandybird-rc-e0whp    1/1          Running       0          35m
+expandybird-rc-zdp8w    1/1          Running       0          35m
+manager-rc-bl4i4        1/1          Running       0          35m
+resourcifier-rc-21clg   1/1          Running       0          35m
+resourcifier-rc-i2zhi   1/1          Running       0          35m
+NAME                    CLUSTER-IP   EXTERNAL-IP   PORT(S)    AGE
+expandybird-service     10.0.0.248   <none>        8081/TCP   35m
+manager-service         10.0.0.49    <none>        8080/TCP   35m
+resourcifier-service    10.0.0.184   <none>        8082/TCP   35m
+NAME                    DESIRED      CURRENT       AGE
+expandybird-rc          2            2             35m
+manager-rc              1            1             35m
+resourcifier-rc         2            2             35m
 ```
 
-If you see expandybird-service, manager-service, resourcifier-service, and
-expandybird-rc, manager-rc and resourcifier-rc with pods that are READY, then DM
-is up and running!
+If you see expandybird, manager and resourcifier services, as well as expandybird, manager and resourcifier replication controllers with pods that are READY, then Helm is up and running!
 
 ## Using Helm
 
-Run a Kubernetes proxy to allow the dm client to connect to the cluster:
+Run a Kubernetes proxy to allow the Helm client to connect to the remote cluster:
 
 ```
-kubectl proxy --port=8001 --namespace=dm &
+kubectl proxy --port=8001 &
 ```
 
-## Uninstalling Helm from Kubernetes
-
-You can uninstall Helm using the same configuration:
+Configure the HELM_HOST environment variable to let the local Helm client talk to the Helm manager service running in your remote Kubernetes cluster using the proxy.
 
 ```
-helm server uninstall
+export HELM_HOST=http://localhost:8001/api/v1/proxy/namespaces/helm/services/manager-service:manager
 ```
 
 ## Installing Charts
 
-To quickly deploy a chart, you can use the Helm command line tool:
+To quickly deploy a chart, you can use the Helm command line tool.
+
+Currently here is the step by step guide.
+
+First add a respository of Charts used for testing:
+
+```
+$ helm repo add kubernetes-charts-testing gs://kubernetes-charts-testing
```

Comment:
should be `./bin/helm repo add ...` to keep consistent with the first few steps at the top